packages/catalog-client/src/types/api.ts

```diff
@@ -257,6 +251,23 @@ export type AddLocationRequest = {
   dryRun?: boolean;
 };
 
+/**
+ * Used for filtering entities that match the given patterns.
+ *
+ * @public
+ */
+export type EntitiesFilter =
+  | Record<string, string | symbol | (string | symbol)[]>[]
+  | Record<string, string | symbol | (string | symbol)[]>
+  | undefined;
+
```

Comment:
EntitiesFilter should only allow this type:
```suggestion
  export type EntitiesFilter = Record<string, (string | symbol)[]>[];
```

Reply:
Actually `EntitiesFilter ` could be removed in favour of `EntityFilterQuery`. Gone.